omod/src/main/webapp/resources/htmlforms/Covid19-Dischargeform.xml

```diff
@@ -0,0 +1,777 @@
+                                            </div>
+                                    </div>
+                                </div>
+                                <div class="card">
+                                    <div class="card-body">
+                                        <div class="row">
+                                            <div class="col-md-2">
+                                                <label>Hydroxychloroquine</label>
+                                                <obs conceptId="166348" answerConceptIds="90003,90004" answerLabels="Yes, No"
+                                                     style="radio" class="horizontal" id="hydroxychloroquine"/>
+                                            </div>
+
+                                        </div>
+                                    </div>
+                                </div>
+                                <div class="card">
+                                    <div class="card-body">
+                                        <div class="row">
+                                            <div class="col-md-2">
+                                                <label>Antiviral Given</label>
+                                                <obs conceptId="166351" answerConceptIds="90003,90004,90001" answerLabels="Yes, No, Unknown"
+                                                     style="radio" class="horizontal" id="antiviral"/>
+                                            </div>
+                                            <div class="col-md-8">
+                                                <label>Antivirals</label>
+                                                <div class="row">
+                                                    <obs id="ribavirin" conceptId="166351" answerConceptId="166350" answerLabel="Ribavirin" style="checkbox" class="col-md-2"/>
+                                                    <obs id="lopinavir-ritonavir" conceptId="166351" answerConceptId="794" answerLabel="Lopinavir/Ritonavir" style="checkbox" class="col-md-3"/>
+                                                    <obs id="neuraminidase-inhibitor" conceptId="166351" answerConceptId="166349" answerLabel="Neuraminidase Inhibitor" style="checkbox" class="col-md-4"/>
+                                                </div>
+                                            </div>
+                                        </div>
+                                    </div>
+                                </div>
+                                <div class="card">
+                                    <div class="card-body">
+                                        <div class="row">
+                                            <div class="col-md-4">
+                                                <label>Corticosteroid</label>
+                                                <obs conceptId="166353" answerConceptIds="90003,90004,90001" answerLabels="Yes, No, Unknown"
```

Comment:
Am going to create concept for Corticosteroid with answers (Yes, No, Unknown) to distinguish it from Corticosteroids (166353)

Reply:
waiting for the concept id